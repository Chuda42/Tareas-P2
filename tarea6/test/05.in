# Implementar distancia
hacerVecinos 5 2 25
distancia 2 5
distancia 5 2
hacerVecinos 5 8 58
distancia 8 5
distancia 5 8
hacerVecinos 9 1 91
distancia 1 9
distancia 9 1
hacerVecinos 2 10 210
distancia 10 2
distancia 2 10
Fin
