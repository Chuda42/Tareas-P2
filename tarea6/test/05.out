1>#  Implementar distancia.
2>Nuevos vecinos: 5 2 25.00.
3>25.00
4>25.00
5>Nuevos vecinos: 5 8 58.00.
6>58.00
7>58.00
8>Nuevos vecinos: 9 1 91.00.
9>91.00
10>91.00
11>Nuevos vecinos: 2 10 210.00.
12>210.00
13>210.00
14>Fin.
